app/views/admin/custom_organisations/index.html.erb

```diff
@@ -16,19 +17,20 @@
       <% end %>
 
     <table class="govuk-table">
-    <h2 class="govuk-heading-m">Custom Organistions</h2>
+    <h2 class="govuk-heading-m">Custom Organisations that are already in our register</h2>
       <thead class="govuk-table__head">
         <tr class="govuk-table__row">
           <th class="govuk-table__header" scope="col">Name</th>
           <th class="govuk-table__header" scope="col">Allowed on</th>
+          <th class="govuk-table__header" scope="col"> </th>
         </tr>
       </thead>
       <tbody class="govuk-table__body">
         <% @custom_organisation_names.each do |organisation| %>
-        <tbody class="govuk-table__body">
-          <tr class="govuk-table__row">
+          <tr class="govuk-table__row" id="data-id-<%= organisation.id %>">
```

Comment:
Sorry I think I led you astray here. The standard approach is `data-id="7"` not `id="data-id-7"`

You may as well just use `id=7` to make it easy.